apps/condo/domains/marketplace/schema/Invoice.js

```diff
@@ -318,6 +322,67 @@ const Invoice = new GQLListSchema('Invoice', {
 
             return resolvedData
         },
+        afterChange: async ({ context, operation, originalInput, updatedItem }) => {
+            const userId = updatedItem.client
+            const propertyId = updatedItem.property
+            const unitName = updatedItem.unitName
+            const unitType = updatedItem.unitType
```

Comment:
You can easy single line this
```suggestion
            const { client: userId, property: propertyId, unitName, unitType } = updatedItem
```